### DISCUSSION TITLE
There is an error during installing Python API

### DISCUSSION THREAD

**Discussion by zaeucu** [microsoft/AirSim#3998] [Support Q&A]
1. i use windows 10 64bit, ue 4.27.0, python 3.7
2. Then i want to use airsim python API, and i followed this link https://microsoft.github.io/AirSim/apis/
3. But when i use the "pip install airsim", there is an error like below,
Collecting airsim
  Using cached airsim-1.6.0.tar.gz (19 kB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\cw\anaconda3\envs\python_3.5\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\CW\\AppData\\Local\\Temp\\pip-install-71zru5l0\\airsim_19f1ef58eed34502af9d96209cc3a674\\setup.py'"'"'; __file__='"'"'C:\\Users\\CW\\AppData\\Local\\Temp\\pip-install-71zru5l0\\airsim_19f1ef58eed34502af9d96209cc3a674\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\CW\AppData\Local\Temp\pip-pip-egg-info-qk8_8iol'
         cwd: C:\Users\CW\AppData\Local\Temp\pip-install-71zru5l0\airsim_19f1ef58eed34502af9d96209cc3a674\
    Complete output (12 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\CW\AppData\Local\Temp\pip-install-71zru5l0\airsim_19f1ef58eed34502af9d96209cc3a674\setup.py", line 2, in <module>
        from airsim import __version__
      File "C:\Users\CW\AppData\Local\Temp\pip-install-71zru5l0\airsim_19f1ef58eed34502af9d96209cc3a674\airsim\__init__.py", line 1, in <module>
        from .client import *
      File "C:\Users\CW\AppData\Local\Temp\pip-install-71zru5l0\airsim_19f1ef58eed34502af9d96209cc3a674\airsim\client.py", line 3, in <module>
        from .utils import *
      File "C:\Users\CW\AppData\Local\Temp\pip-install-71zru5l0\airsim_19f1ef58eed34502af9d96209cc3a674\airsim\utils.py", line 208
        logging.error(f"Writing PNG file {filename} failed")
                                                          ^
    SyntaxError: invalid syntax

4. and my console installing airsim 1.5.0 version automatically, but there is same error on "logging.error(f..."

Sure, it can be solved by erasing that code, but is there somebody like me? i found that code used in several versions,

so i want to know why i have that error.

Thanks

**Comment by rajat2004** [comment]
Python 3.5 is being used, not 3.7 - `c:\users\cw\anaconda3\envs\python_3.5\python.exe`
